core/src/main/java/com/linecorp/armeria/client/AbstractClientFactory.java

```diff
@@ -96,18 +98,20 @@
     protected final Scheme validateScheme(URI uri) {
         requireNonNull(uri, "uri");
 
-        final String scheme = uri.getScheme();
-        if (scheme == null) {
-            throw new IllegalArgumentException("URI with missing scheme: " + uri);
-        }
-
-        if (uri.getAuthority() == null) {
+        if (Objects.isNull(uri.getAuthority())) {
```

Comment:
`Objects.isNull()` could be replaced with ` == null` for simplicity.

Reply:
Done. Habit of working on my code, where we prefer isNull() for readability :-)